docs/orchestrating-elastic-stack-applications/elasticsearch/advanced-node-scheduling.asciidoc

```diff
@@ -245,6 +245,13 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.annotations['topology.kubernetes.io/zone']
+        initContainers:
```

Comment:
I think this should only be done if some secure settings have been specified, otherwise this init container will try to start Elasticsearch itself (or whatever the entry point for the Elasticsearch container is).I would comment that part and explain in which cases it should be uncommented.

I also wonder if we should add a warning in both the secure settings section and here.

Reply:
Ok, I've commented this section out, and added some notes for when to uncomment it, and added a warning in the secure settings documentation as well.   Verifying generated docs work as expected, as I'm using some new (to me) [cross-doc references](https://docs.asciidoctor.org/asciidoc/latest/macros/inter-document-xref/)